statannot/statannot.py

```diff
@@ -27,6 +27,9 @@ def stat_test(
 ):
     """Get formatted result of two sample statistical test.
 
+    EDIT 16.11.2020 - Veronika Brumovska
+    - implementing ANOVA (line 84)
```

Comment:
I don't think this was ever done in this repo, but it would be great to be logged.
I wouldn't put that here though. Maybe we could suggest a changelog or acknowledgement file dedicated to contributions ?